plugins/destination/gcs/client/spec/schema.go

```diff
@@ -133,14 +162,21 @@ func (s Spec) JSONSchemaExtend(sc *jsonschema.Schema) {
 						properties.Set("path", &jsonschema.Schema{
 							Not: &jsonschema.Schema{
 								Type:    "string",
-								Pattern: `^.*{{.*}}.*$`,
+								Pattern: `^.*\{\{.*\}\}.*$`,
```

Comment:
This regex works on Go apperatnly but fails in JavaScript, so we need to escape the brackets